C23_WMDE_Desktop_EN_04

Abban opened the pull request:
- Based on the CTRL of test 03. Both banners have fallback and new live time text.
- VAR has an updated version of the payment bubble radio fields.

Ticket: https://phabricator.wikimedia.org/T352634

Abban committed:
Archive C23_WMDE_Desktop_EN_04

Ticket: https://phabricator.wikimedia.org/T352634

Abban committed:
Prepare campaign for C23_WMDE_Desktop_EN_04

Based on the CTRL of test 03. Both banners have fallback and
new live time text.

Ticket: https://phabricator.wikimedia.org/T352634

Abban committed:
Implement VAR for C23_WMDE_Desktop_EN_04

VAR has an updated version of the payment bubble
radio fields.

Ticket: https://phabricator.wikimedia.org/T352634

gbirke committed:
Restart Campaign C23_WMDE_Desktop_EN_04

The bug in createFallbackDonationURL interefred with the A/B test
tracking so we have to restart.

Ticket: https://phabricator.wikimedia.org/T352634

moiikana commented on banners/english/styles/MainBannerVar.scss:
```diff
@@ -0,0 +1,45 @@
+
+$banner-height: 357px !default;
+$form-width: 300px !default;
+
```
this file should probably be named MainBanner(Ctrl)

Abban committed:
Add styles for fallback late progress to C23_WMDE_Desktop_DE_20

The Fallback styles needed adjustment to fit in the late progress
bar on the small banner.

Ticket: https://phabricator.wikimedia.org/T352634
